tools/jsonstats.cpp

```diff
@@ -180,18 +181,34 @@ stat_t simdjson_compute_stats(const simdjson::padded_string &p) {
 }
 
 int main(int argc, char *argv[]) {
-  int myoptind = 1;
-  if (myoptind >= argc) {
-    std::cerr << "Reads json, prints stats. " << std::endl;
-    std::cerr << "Usage: " << argv[0] << " <jsonfile>" << std::endl;
+  std::string progName = "jsonstat";
+  std::string progUsage = "Reads json, prints stats.\n";
+  progUsage += argv[0];
+  progUsage += " <jsonfile>";
+
+  cxxopts::Options options(progName, progUsage);
+
+  options.add_options()
+    ("h,help", "Print usage.")
+    ("f,file", "File name.", cxxopts::value<std::string>())
+  ;
+
+  options.parse_positional({"file"});
+  auto result = options.parse(argc, argv);
+
+  if(result.count("help")) {
+    std::cerr << options.help() << std::endl;
     exit(1);
```

Comment:
Consider turning all of the exit(1) into return EXIT_FAILURE, for improved portability.

Reply:
Done!

Reply:
+1